docs/how-to-guides/use-a-custom-timestamp-column.md

```diff
@@ -0,0 +1,213 @@
+# How to use a custom timestamp column
+
+## Context
+
+You have events that have a timestamp attribute. You want to do time-related processing over them and want ksqlDB to use those timestamps for processing. Because ksqlDB defaults to using wall clock time, you need to tell ksqlDB where to find the timestamps within the events. This is known as *processing-time* (using the wall clock) and *event-time* (using timestamps within events).
```

Comment:
> Because ksqlDB defaults to using wall clock time

The way that I interpreted this isn't true - we use the default timestamp column in the message, not the time we processed the message. The reason that `INSERT INTO` makes it look like wall clock time is because it sets the timestamp field to be the wall clock time if not specified. The `timestamp` column allows you to use something other than the default timestamp in the kafka message

Reply:
Ohh, nice catch. I've updated the text. Let me know if that reads clearly.